prime-dotnet-webapi/HttpClients/LDAP Api/LdapClient.cs

```diff
@@ -42,7 +43,13 @@ public async Task<string> GetUserAsync(string username, string password)
                 if (response.IsSuccessStatusCode)
                 {
                     var successResponse = JsonConvert.DeserializeObject<GisUser>(responseJsonString);
```

Comment:
Why de-serialize to `GisUser` then map to `LdapResponseKeys` when you can just de-serialize to `LdapResponseKeys` or return the `GisUser`?